src/database/seed.sql

```diff
@@ -0,0 +1,28 @@
+BEGIN;
+
+INSERT INTO users (id, name) VALUES
+(1, 'mark'),
```

Comment:
The id shouldn't be required here as the field is set to autoincrement. Not sure if this will throw a sqlite error.